pkg/build/controller/controller.go

```diff
@@ -220,7 +223,7 @@ func (bc *BuildPodController) CancelBuild(build *buildapi.Build, pod *kapi.Pod)
 		return err
 	}
 
-	glog.V(2).Infof("Build %s was successfully cancelled.", build.Name)
+	glog.V(2).Infof("Build %s/%s was successfully cancelled.", build.Namespace, build.Name)
```

Comment:
Build controllers should be at v(4) for almost all informational messages.  V(2) should be reserved for exceptional cases.

> On May 25, 2015, at 10:53 AM, Jakub Hadvig <notifications@github.com> wrote:
> 
> In pkg/build/controller/controller.go:
> 
> > @@ -220,7 +223,7 @@ func (bc *BuildPodController) CancelBuild(build *buildapi.Build, pod *kapi.Pod)
> >  		return err
> >  	}
> >  
> > -	glog.V(2).Infof("Build %s was successfully cancelled.", build.Name)
> > +	glog.V(2).Infof("Build %s/%s was successfully cancelled.", build.Namespace, build.Name)
> I think its better to leave that verbosity level as it is(also for the comments mentioned below) so the user would see different set of output on different log level of the client.
> 
> —
> Reply to this email directly or view it on GitHub.
>